xml/admin_ceph_gateway.xml

```diff
@@ -2045,17 +2045,6 @@ GET /<replaceable>BUCKET</replaceable>?mdsearch
        </para>
```

Comment:
Drop the last sentence here, that was needed only to reference the now deleted sentence

Reply:
Also the title should be just "resharding the bucket index" drop pool